h/static/scripts/tests/controllers/tooltip-controller-test.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+var TooltipController = require('../../controllers/tooltip-controller');
+
+// helper to dispatch a native event to an element
+function sendEvent(element, eventType) {
```

Comment:
We upgraded to PhantomJS 2.x since this was originally written. PhantomJS 2.x _does_ support the _Event_ constructor, so you can just do:

```js
element.dispatchEvent(new Event('mouseover'))
```

For example